DAS/simulator.py

```diff
@@ -242,9 +242,9 @@ def run(self):
                 self.validators[i].updateStats()
             trafficStatsVector.append(trafficStats)
 
-            missingSamples, sampleProgress, nodeProgress, validatorProgress = self.glob.getProgress(self.validators)
-            self.logger.debug("step %d, arrived %0.02f %%, ready %0.02f %%, validated %0.02f %%"
-                              % (steps, sampleProgress*100, nodeProgress*100, validatorProgress*100), extra=self.format)
+            missingSamples, sampleProgress, nodeProgress, validatorAllProgress, validatorProgress = self.glob.getProgress(self.validators)
+            self.logger.info("step %d, arrived %0.02f %%, ready %0.02f %%, validatedall %0.02f %%, , validated %0.02f %%"
```

Comment:
Do you want to keep this as info? Or was it just for debugging?

Reply:
it should be debug, good catch. I thought I've already changed it, but seemingly not :-)